x-pack/plugins/apm/ftr_e2e/cypress/integration/service_overview/time_comparison.spec.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import url from 'url';
+import archives_metadata from '../../fixtures/es_archiver/archives_metadata';
+const { start, end } = archives_metadata['apm_8.0.0'];
+
+const baseUrl = url.format({
+  pathname: '/app/apm/services/opbeans-java/overview',
+  query: { rangeFrom: start, rangeTo: end },
+});
+
+const apisToIntercept = [
+  {
+    endpoint: '/api/apm/services/opbeans-java/transactions/charts/latency',
+    as: 'latencyChartRequest',
+  },
+  {
+    endpoint: '/api/apm/services/opbeans-java/throughput',
+    as: 'throughputChartRequest',
+  },
+  {
+    endpoint: '/api/apm/services/opbeans-java/transactions/charts/error_rate',
+    as: 'errorRateChartRequest',
+  },
+  {
+    endpoint:
+      '/api/apm/services/opbeans-java/transactions/groups/detailed_statistics',
+    as: 'transactionGroupsDetailedRequest',
+  },
+  {
+    endpoint: '/api/apm/services/opbeans-java/error_groups/detailed_statistics',
+    as: 'errorGroupsDetailedRequest',
+  },
+  {
+    endpoint:
+      '/api/apm/services/opbeans-java/service_overview_instances/detailed_statistics',
+    as: 'instancesDetailedRequest',
+  },
+];
+
+describe('Service overview: Time Comparison', () => {
+  it('enables by default the time comparison feature with Last 24 hours selected', () => {
+    cy.visit(baseUrl);
+    cy.url().should('include', 'comparisonEnabled=true&comparisonType=day');
+  });
+
+  it('toggles comparison off disables select box and APIs are called without comparison time range', () => {
+    apisToIntercept.map(({ endpoint, as }) => {
+      cy.intercept('GET', endpoint).as(as);
+    });
+    cy.visit(baseUrl);
+    cy.contains('opbeans-java');
+
+    // Comparison is enabled by default
```

Comment:
Would it make more sense to group this with the previous item?

Reply:
What do you mean @smith ?

Reply:
The previous `it` block, which tests if comparisons are enabled.